### PR TITLE
Grid/reset action

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The reset action wasn't working on Grids, this is now fixed
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/prestafony-project/issues/43
| How to test?  | Access `http://localhost/admin-dev/index.php/configure/advanced/logs/` and play with Reset action (clear the cache first for javascripts file)

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9286)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/app/utils/reset_search.js [hunk 1]
```diff
@@ -29,8 +29,10 @@
 
 const $ = global.$;
 
-const init = function resetSearch(url) {
+const init = function resetSearch(url, redirectUrl) {
     $.post(url);
+    console.log(redirectUrl);
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/app/utils/reset_search.js]: tsssss

Comment (mickaelandrieu) [reply admin-dev/themes/new-theme/js/app/utils/reset_search.js]: wasn't me, I saw you appending my commit OMG :trollface:

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/app/utils/reset_search.js]: It's @tomlev fault? :D

**Comment by mickaelandrieu** [comment]
ping @PierreRambaud, waiting for QA or? :)

**Review by PierreRambaud — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks everyone!

**Pull request merged**